madlib.json

```diff
@@ -1,12 +1,21 @@
 {
   "name": "instigator",
```

Comment:
We still need to define how we want to go on, but so far I've always been using capitalized names for modules and packages. I know that may be a shift from JS, but also with types and constructors having capitalized names and stuff I think there's a need for high level stuff to be capitalized.

Reply:
Yeah, I don't feel strongly enough here to argue about it. But that leaves us open to `_XxX_CoolStuff_XxX_` names too right?